File: frontend/src/models/YouTube.ts
export class YouTube {
    id: number;
    title: string;
    url: string;
    time: string;
    constructor(title: string, id: number, url: string, time: string) {
      this.title = title;
      this.id = id;
      this.url = url;
      this.time = time;
    }
}

File: telescreen/src/main/java/ru/panfio/telescreen/model/YouTube.java
package ru.panfio.telescreen.model;

import com.fasterxml.jackson.annotation.JsonAlias;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import lombok.Data;
import lombok.NoArgsConstructor;
import ru.panfio.telescreen.util.IsoInstantDeserializer;
import ru.panfio.telescreen.util.IsoInstantSerializer;

import java.time.Instant;

@Data
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class YouTube {
    private Long id;
    private String title;
    @JsonAlias("titleUrl")
    private String url;
    @JsonSerialize(using = IsoInstantSerializer.class)
    @JsonDeserialize(using = IsoInstantDeserializer.class)
    private Instant time;
}
